build-system/scripts/query_manifest

```diff
@@ -14,7 +14,7 @@ fi
 
 function get_deps {
   local TYPE=$(yq -r ".\"$1\".dependencies | type" $MANIFEST)
-  if [ "$TYPE" == "!!str" ]; then
+  if [ "$TYPE" == "!!str" ] || [ "$TYPE" == "string" ]; then
```

Comment:
I think I deliberately didn't support this alternative yq due to other edge cases, and the principle of "one way".
Although I honestly now don't remember.
Are the differing versions npm vs apt? I think that might be it.

Anyway i can't remember why I didn't support both, so I don't have a good argument to not.

Reply:
I dug deeper and it turns out I was using yq v3 instead of v4, huh. I agree on the general sentiment of "doing things one way", so I'm rolling back this change.